.github/workflows/build.yaml

```diff
@@ -21,6 +21,9 @@ jobs:
             matrix:
                 php: [7.4, 8.0, 8.1]
```

Comment:
Probably makes sense to add 8.2, 8.3. 8.4 as well, since this is more of standalone package.